apps/condo/domains/user/utils/redisGuard.js

```diff
@@ -2,6 +2,22 @@ const conf = require('@core/config')
 const IORedis = require('ioredis')
```

Comment:
move to apps/condo/domains/user/utils/serverScheema/guards.js please

Reply:
done